esp-metadata/devices/esp32c6.toml

```diff
@@ -251,6 +252,12 @@ input_signals = [
     { name = "SPID",                id = 122 },
     { name = "SPIHD",               id = 123 },
     { name = "SPIWP",               id = 124 },
+
+    { name = "SD_CMD" },
+    { name = "SD_DATA0" },
+    { name = "SD_DATA1" },
+    { name = "SD_DATA2" },
+    { name = "SD_DATA3" },
```

Comment:
These should follow the naming in the TRM, i.e. `SDIO_*`. I'm adding these signals among others in a separate PR, but in case you win that race, please rename these :)

Reply:
I rebased on your changes, and updated the names in ESP32 to be consistent with ESP32C6. This just makes it easier to use in the SDIO driver, let me know if you want me to do something different.